src/System.Runtime.Extensions/src/System/Environment.Unix.cs

```diff
@@ -51,7 +52,7 @@ private static string ExpandEnvironmentVariablesCore(string name)
             }
             result.Append(name.Substring(lastPos));
```

Comment:
You can replace `result.Append(name.Substring` will `name.AsSpan` to void unnecessary allocations while you are on it.

Reply:
+

Reply:
there's still one left in line 53